tests/console/commands/env/conftest.py

```diff
@@ -25,8 +25,8 @@ def venv_name(app: PoetryTestApplication) -> str:
 
 
 @pytest.fixture
-def venv_cache(tmp_dir: str) -> Path:
-    return Path(tmp_dir)
+def venv_cache(tmp_path: Path) -> Path:
```

Comment:
if TYP_CHECKING is False, it wont fail?

Reply:
`Path` is not used outside of type annotations and `TYPE_CHECKING` is false at runtime when tests are run.